src/storage/sftp/helper.c

```diff
@@ -32,7 +32,9 @@ storageSftpHelper(const unsigned int repoIdx, const bool write, StoragePathExpre
             .pathExpressionFunction = pathExpressionCallback, .modeFile = STORAGE_MODE_FILE_DEFAULT,
             .modePath = STORAGE_MODE_PATH_DEFAULT, .keyPub = cfgOptionIdxStrNull(cfgOptRepoSftpPublicKeyFile, repoIdx),
             .keyPassphrase = cfgOptionIdxStrNull(cfgOptRepoSftpPrivateKeyPassphrase, repoIdx),
-            .hostFingerprint = cfgOptionIdxStrNull(cfgOptRepoSftpHostFingerprint, repoIdx)));
+            .hostFingerprint = cfgOptionIdxStrNull(cfgOptRepoSftpHostFingerprint, repoIdx),
+            .sftpStrictHostKeyChecking = cfgOptionIdxStrId(cfgOptRepoSftpStrictHostKeyChecking, repoIdx),
+            .sftpKnownHosts = cfgOptionIdxLst(cfgOptRepoSftpKnownHosts, repoIdx)));
 }
```

Comment:
Passing known hosts around as a variant list is a bit awkward -- perhaps use strLstNewVarLst() to transform it into a string list?

Reply:
Switched to passing a StringList in 
 https://github.com/pgbackrest/pgbackrest/pull/2125/commits/0716d6fa556caac34864f9a370575ae643c53f01